services/wireguard/endpoint/kernelspace/client.go

```diff
@@ -73,6 +74,14 @@ func (c *client) AddPeer(iface string, peer wg.PeerInfo) error {
 		if err != nil {
 			return err
 		}
+		if endpoint != nil {
```

Comment:
can we reduce nesting in this method in general? something like 
```golang
if peer == nil {
  // return
}
rest of the code here
```

Reply:
Figured out that we don't need this check at all. Looks like some leftover. 👍